### ISSUE TITLE
PermissionsEX + BungeeCord

### ISSUE THREAD

**Issue by Artemon256** [SkinsRestorer/SkinsRestorerX#121]
### what SkinsRestorer version are you using? (make sure its latest!)
13.6.1

### Describe the problem in a few short sentences
Nobody can't set their skins, even admin, who has '*' permission.

### How can we reproduce the problem?
I have BungeeCord+Spigot and PermissionsEX configured on both of them. PEX uses mysql database, same thing goes for SkinsRestorer

### What is your Spigot/BungeeCord/Sponge version?
BungeeCord 1.12.2

**Comment by xknat** [comment]
@Artemon256 its because you need to set permissions on bungee side, make sure to read our install again in the wiki

**Comment by Artemon256** [comment]
Bungee has PermissonsEX configured to use mysql database which is common for all servers (I have only one server at the moment), and PEX on server is configured in the same way. Am I wrong with it?

**Comment by xknat** [comment]
@Artemon256 i would like to fall back on my previous comment in which i stated to read the wiki carefully and follow ALL steps in doing so:
https://github.com/SkinsRestorer/SkinsRestorerX/wiki/Installing-SkinsRestorer#installing-Bungee
Let me know regarding each individual step if its performed / done

**Comment by Artemon256** [comment]
I have followed every step of that instruction, what do I have now:
1. spigot.yml has bungeecord: true option
2. On spigot server there is SkinsRestorer plugin, it's visible in /plugins command, but folder plugins/SkinsRestorer is empty
3. BungeeCord server has SkinsRestorer plugin, it's config folder has files. config.yml has next entries:
`SkinWithoutPerm: false`
`Permissions:
  NewPermissions: true`
`MySQL:
  Enabled: true
  Host: [my hostname]
  Port: 3306
  Database: [my db]
  Username: [db user]
  Password: [password]
  SkinTable: 'Skins'
  PlayerTable: 'Players'`
4. BungeeCord and Spigot have PermissionsEX installed and configured to use one MySQL database, so their configs are same.
5. I am admin and I have '*' permission.
6. When I type /skin [SkinName] I receive "[SkinsRestorer] I'm sorry, but you do not have permission to perform this command".

Thank you in advance.

**Comment by McLive** [comment]
Hi!

You might reach out to https://github.com/aikar/commands/issues since we use their permission annotations in our SR plugin.

**Comment by Artemon256** [comment]
I don't understand how does it relate to SR. What can I write in their issues section?

**Comment by McLive** [comment]
It seems like it doesn't work with permission systems on bungeecord. I'm not really sure. I'll ask them

**Comment by Artemon256** [comment]
Thank you

**Comment by McLive** [comment]
Which plugin have you installed on BungeeCord? I don't think there is a PermissionsEx for bungeecord. You'd need something like LuckPerms or BungeePEXBridge there.

**Comment by Artemon256** [comment]
Thank you! Setting up BungeePEXBridge did the trick.

**Comment by aljaxus** [comment]
For future reference;
As of now PermissionsEX is a plugin for bukkit only (also works on spigot, paper, and other forks).
Permissions on Bukkit are not the same thing as permissions on Bungeecord. If you are using Bungeecord, the permissions are totally separate and independent of bukkit/spigot (sub-server) permissions.

To sync permissions, ranks (and more) across all of your servers, you can use [LuckPerms plugin](https://luckperms.github.io), which, as of right now, works on all major platforms (bukkit, spigot, paper, sponge, bungeecord, ...) and is being actively developed and improved. It also supports importing data from other permission plugins (easy to switch). For more info about LuckPerms, please look at [their wiki](https://github.com/lucko/LuckPerms/wiki).

tldr; Spigot permissions are not the same as Bungeecord permissions. Please start using LuckPerms (or any other **updated** permissions plugin).
